blocks/quote-carousel/quote-carousel.js

```diff
@@ -109,6 +109,7 @@ async function getContent(block) {
 export default async function decorate(block) {
   const blockId = crypto.randomUUID();
   const content = await getContent(block);
+  const title = block.querySelector('div > div > div > div > div').textContent;
```

Comment:
This should be based on row, not column.

First row is title, second row is the link to the sheet with the quotes.